pkg/deployments/deployments.go

```diff
@@ -117,6 +118,30 @@ func (provider *DefaultImageProvider) getImageForDeployGroup(deploymentGroup str
 	return deployment.DockerImage, nil
 }
 
+func GetPaastaGitShaFromDockerURL(dockerUrl string) (string, error) {
+	image := strings.Split(dockerUrl, "/")
+	if len(image) != 2 {
+		return "", fmt.Errorf(
+			"Failed to extract paasta git sha from url: %s",
+			dockerUrl,
+		)
+	}
+	gitShaStrings := strings.Split(image[1], "-")
+	if len(gitShaStrings) != 2 {
+		return "", fmt.Errorf(
+			"Failed to extract paasta git sha from url: %s",
+			dockerUrl,
+		)
+	}
+	if len(gitShaStrings[1]) < 8 {
```

Comment:
why 8? i think 7 was long time default for git.

Reply:
https://github.com/Yelp/paasta/blob/530354023d5851d68d8648d7285eb8501f8c2f27/paasta_tools/utils.py#L3025

Who knows: https://github.com/Yelp/paasta/blob/530354023d5851d68d8648d7285eb8501f8c2f27/paasta_tools/utils.py#L3025

Reply:
but actually am I off by one anyway? shouldn't it be 9?